hack/test-go.sh

```diff
@@ -121,7 +121,7 @@ done
 shift $((OPTIND - 1))
 
 # Use eval to preserve embedded quoted strings.
-eval "goflags=(${KUBE_GOFLAGS:-})"
```

Comment:
yeah, `unit_test` might be better, to differentiate from things used by integration or e2e tests.